src/transformState.js

```diff
@@ -5,7 +5,34 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'removeProperties':
+        if (typeof (action.keysToRemove) === 'object') {
+          for (const key of action.keysToRemove) {
+            delete state[key];
+          }
+        } else {
+          delete state[action.keysToRemove];
+        }
```

Comment:
if/else statement is redundant here, use only one cycle for getting needed keys and delete properties from state

Reply:
keysToRemove always contains array